Makefile

```diff
@@ -30,6 +30,8 @@ endif
 OS = $(shell go env GOOS)
 ARCH = $(shell go env GOARCH)
 
+OPERATOR_NAMESPACE ?=
+REPLACES ?=
```

Comment:
Please move this definition immediately after `VERSION` definition (line2) as they are tightly coupled. Also, this should have a default value of the current version we are replacing

Reply:
done.